community/kernel/src/main/java/org/neo4j/kernel/TransactionEventHandlers.java

```diff
@@ -49,7 +48,7 @@
  */
 @Deprecated
 public class TransactionEventHandlers
-    implements Lifecycle, TransactionHook<TransactionEventHandlers.TransactionHandlerState>
+        implements Lifecycle, TransactionHook<TransactionEventHandlers.TransactionHandlerState>
```

Comment:
An observation for next time - can we (all) try to avoid spurious re-formatting in PRs that add a lot of additional stuff as well? It lowers the signal/noise ratio, and makes it hard to review.